dist/assembly.xml

```diff
@@ -136,6 +136,19 @@
       <directoryMode>0755</directoryMode>
     </dependencySet>
 
+    <!-- put the agent installer to the same directory as previous artefact from the same reason -->
+    <dependencySet>
+      <outputDirectory>${hawkular.dist.zip.root.dir}/standalone/configuration</outputDirectory>
+      <useProjectArtifact>false</useProjectArtifact>
+      <includes>
+        <include>org.hawkular.agent:hawkular-wildfly-agent-installer:jar</include>
+      </includes>
+      <outputFileNameMapping>agent-installer.${artifact.extension}</outputFileNameMapping>
```

Comment:
this needs to be called "hawkular-wildfly-agent-installer.${artifact.extension}" - and I would go so far as to put the version string in there so we know exactly what is in there just by looking at the name, but that is less of an issue. But for sure the name needs to be "hawkular-wildfly-agent-installer"